app/templates/index.hbs

```diff
@@ -1,36 +1,69 @@
 {{outlet}}
-<div class="au-u-padding-top-large au-u-padding-left au-u-padding-right">
-  <div class="au-o-grid">
-    <div class="au-o-grid__item au-u-1-2@medium">
-      <h1 class="au-u-h1 au-u-margin-bottom">Test loket voor meldingen</h1>
-    </div>
-    <div class="au-o-grid__item au-u-1-2@medium au-u-text-right">
-      <LinkTo @route="forms.new" class="au-c-button" type="button">Maak nieuw formulier aan</LinkTo>
-    </div>
-  </div>
-
-  <DataTable
-          @content={{this.model}}
-          @isLoading={{this.isLoadingModel}}
-          @filter={{this.filter}}
-          @sort={{this.sort}}
-          @page={{this.page}}
-          @size={{this.size}}
-          as |t|
-  >
+{{#let (t "main.title")
+       (t "main.create-submission")
+       (t "submission.unit")
+       (t "submission.type")
+       (t "submission.date")
+       (t "submission.creator")
+       (t "submission.modifier")
+       (t "submission.sent")
+       (t "submission.status")
+       (t "submission.action")
+       (t "submission.created-automatic")
+       (t "submission.details")
```

Comment:
Nitpick, but I find this strange to see and it looks like a pain to maintain since it's order dependent. Only `createdAutomatic` seems used twice, but even then I would just write it twice instead of forcing an extra level of indentation 😄.

Reply:
I really despise the way this looks. The let is needed because for some reason the DataTable and those translation don't want to mix.

Indentation? Every item on a new line makes it more readable, no? Also much easier to edit for me using block editing.

Reply:
The reason it doesn't work in the current code is because the block param is also called `t` so it shadows the helper. If that is renamed to something like `table` it will work again.

With indentation I meant that the `let` content is indented an extra level.

Reply:
Ah another one of those "how did I not see that before" things. I'll clean this mess up.

And the indentation is because block content is always indented for me. If blocks, unless blocks, HTML block content, ... so also the let block content.